app/controllers/events_controller.rb

```diff
@@ -4,6 +4,14 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
```

Comment:
Hm, @bgeuken what do you think about introducing controller namespaces as I don't like introducing participants here.

http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/